graphql.md

```diff
@@ -0,0 +1,54 @@
+# GraphQL Development
+
+Our teams have extensive knowledge in developing GraphQL solutions and recomending number of packages depending on your
+use cases and languages.
+
+## Building GraphQL API in from scratch
+
+When building GraphQL API from scratch we recomend using reference GraphQL-js reference implementation which was 
+proven to be the most performant and have continous support from community. Entire development is currently backed by Linux foundation.
+
+### GraphQL Server
+
+For GraphQL Server we recomend using GraphQL-Express for exposing GraphQL APIs over the network 
+
+https://graphql.org/graphql-js/running-an-express-graphql-server
+
+For building GraphQL schema we recomend using GraphQL-Tools:
+
+https://github.com/ardatan/graphql-tools
+
+Developers can use top level database query languages. 
+We recomend using Knex(http://knexjs.org/) for performing queries from GraphQL to relational databases.
```

Comment:
Any reason to recommend Knex.js? For plain SQL queries the user writes would it be better to use plain SQL drivers?

Reply:
Idea here is to promote packages that are widely used/battle tested. In our case that is Knex.js but that is probably different category overal.